rules/native_binary.bzl

```diff
@@ -20,14 +20,13 @@ do, but they run the wrapped binary directly, instead of through Bash, so they
 don't depend on Bash and work with --shell_exectuable="".
```

Comment:
Nit: take the opportunity to fix misspelling? `--shell_executable`, not exectuable

Reply:
Done